src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1583,7 +1583,7 @@ public static EmptyStacktraceException exception(@Nullable Throwable cause, fina
 					logEx("Here is full list of them:");
 					StringBuilder pluginsMessage = new StringBuilder();
 					for (PluginDescriptionFile desc : pluginPackages.values()) {
-						pluginsMessage.append(desc.getName());
+						pluginsMessage.append(desc.getName()).append("-").append(desc.getVersion());

```

Comment:
```suggestion
						pluginsMessage.append(desc.getName()).append(" ").append(desc.getVersion());
```
:/

Reply:
In my opinion the space seems odd, what do you think about square brackets?
`skript-mirror [v2.0.0-SNAPSHOT] (website)`

Reply:
skript-mirror 2.0.0-SNAPSHOT (https://github.com/btk5h/skript-mirror) TuSKe 1.8.2-Pikachu-Patch-3 (github.com/Tuke-Nuke/TuSKe) SK-NBeeT 2.7.2 (https://github.com/ShaneBeee/Sk-NBeeT)

this looks pretty good

Reply:
IMO space is good, too.

Reply:
Better would be directly using the PluginDescriptionFile#getFullName method, which is in the format `skript-mirror v2.0.0-SNAPSHOT`

```suggestion
						pluginsMessage.append(desc.getFullName());
```

Reply:
ahh good call, I'll make the change now.